robottelo.properties.sample

```diff
@@ -113,13 +113,18 @@ ssh_key=
 # rhel6_os=http://example.com/yum/rhel6-os/
 # rhel7_os=http://example.com/yum/rhel7-os/
 
+# RHEL 8 repos
+# rhel8_os=baseos=http://example.com/rhel-8/BaseOS/x86_64/os/,appstream=http://example.com/rhel-8/AppStream/x86_64/os/
+
+
 # If capsule and satellite tools repositories available related packages will
 # be pulled from there instead of using the CDN channel. These information is
 # more suited to be used for downstream, downstream-iso and zstream builds.
 # capsule_repo=http://capsule/repo
 # sattools_repo=
 #   rhel6=http://sattools/repo/el6,
 #   rhel7=http://sattools/repo/el7,
+# swid_tools_repo=https://example.com/swid-rhel-8.repo
```

Comment:
I think I will add a comment explaining more and keep it here. At this moment of time, There is no timeline yet to decide when this will get merged in RHEL8.2. This will take time.